R/FilterStateChoices.R

```diff
@@ -530,6 +529,12 @@ ChoicesFilterState <- R6::R6Class( # nolint
                     private$varname,
                     private$dataname
                   ))
+                  if(is.null(input$selection) && isFALSE(private$multiple)) {
+                    selection_state <- private$get_selected()[1]
+                  } else {
+                    selection_state <- as.logical(input$selection)
+                  }
+
```

Comment:
is `selection_state` used at all?

Reply:
My mistake. It appears that the code in question was not utilized but was instead residual code from my testing.